core/src/main/scala/com/raphtory/internals/management/PythonEncoder.scala

```diff
@@ -181,10 +197,116 @@ object PythonEncoder {
 }
 
 object PythonInterop {
+  val logger: WrappedLogger = new WrappedLogger(Logger(LoggerFactory.getLogger(this.getClass)))
 
-  def assignId(s: String): Long =
+  def assign_id(s: String): Long =
     GraphBuilder.assignID(s)
 
+  def camel_to_snake(s: String): String =
+    PythonEncoder.camelToSnakeCase(s)
+
+  def testArgs(args: Any): Unit =
+    println(s"testArgs($args)")
+
+  def decode[T](obj: Any): T =
+    (obj match {
+      case obj: java.util.ArrayList[_] => obj.asScala
+      case obj                         => obj
+    }).asInstanceOf[T]
+
+  def get_wrapper_str(obj: Any): String =
+    obj match {
+      case _: Iterable[_]       => "Iterable"
+      case _: Iterator[_]       => "Iterator"
+      case _: GraphPerspective  => "TemporalGraph"
+      case _: Table             => "Table"
+      case _: Accumulator[_, _] => "Accumulator"
+      case _: Vertex            => "Vertex"
+      case _: GraphState        => "GraphState"
+      case _                    => "None"
+    }
+
+  def find_class(name: String): Any = {
+    val runtimeMirror = universe.runtimeMirror(getClass.getClassLoader)
+    val module        = runtimeMirror.staticModule(name)
+    val obj           = runtimeMirror.reflectModule(module)
+    obj.instance
+  }
+
+  def make_varargs(obj: Iterable[Any]): Array[Object] =
```

Comment:
Need some comments or documentary of the method and what is doing

Reply:
done